src/pages/Profile.vue

```diff
@@ -144,34 +140,73 @@ body
     max-width: 800px
 
 .input
-  margin: 20px 10px
+  margin: 10px 10px
+  background-color: #fdf4eb
+  border-radius: 12px
+  border-bottom: 0
+  padding: 10px
 
 .error
   background: RED
   color: WHITE
   padding: 10px 25px
   margin-bottom: 15px
-  border-radius: 19px
+  border-radius: 12px
   font-size: 13px
 
 h2
   text-transform: none
+  font-size: 2.3rem
+  font-weight: bold
+  color: #EF7D18
+
+.profile-header
+  margin-left: 12px
+
+.submit-button
+  margin-left: 10px
+  margin-top: 6px
```

Comment:
Quasar comes with some CSS classes for Paddings and Margins. Those are mobile-ready:
https://quasar.dev/style/spacing#Table-of-permutations

Reply:
Looks like I can't specify the exact amount of margin with those classes

Reply:
Btw the whole site is broken in mobile.

Reply:
Yes, I know. But I still think `px` isn't the right choice.
We could create a Component-Button. Otherwise, we would have individual Buttons on each page, right?
Just experiences I made.

Reply:
We can refactor later